versions.props

```diff
@@ -4,6 +4,7 @@ com.google.guava:guava = 31.1-jre
 com.uber.nullaway:nullaway = 0.10.2
 junit:junit = 4.13.2
 org.assertj:assertj-core = 3.23.1
+org.jetbrains:annotations = 23.0.0
```

Comment:
looking through https://github.com/JetBrains/java-annotations/blob/master/CHANGELOG.md it doesn't seem like there have been any API breaks which is good.